CommunityGuidelines.md

```diff
@@ -0,0 +1,60 @@
+- While some cursing is appropriate in context, refrain from using excessive profanity.
+- Keep conversations in the appropriate channel. Some off-topic discussions are to be expected, but try not to blatantly post non-subject-related content.
+- No not safe for work (NSFW) material is allowed. Breaking this rule will be met with an immediate ban.
+- Keep spam to a minimum:
+  - Do not repeatedly send the same message.
+  - Do not post links to sites/content others would find unacceptable.
+- Advertising or promotion of piracy is forbidden.
+- Staff members (`@Developer` on Discord) have the final say about what is and is not acceptable within the community. Refusals to listen to staff may result in being kicked or banned.
+- HAVE FUN and enjoy all that the community has to offer.
+
+To read our Code of Conduct please refer to [Code of Conduct](CODE_OF_CONDUCT.md).
+ 
+## Team rules
+
+On top of the community rules, the development and moderation team has to keep themselves to a specific set of rules.
+
+### Behavioural rules
+
+- Keep cool to any community member, even if they can be very frustrating to deal with: Do not retaliate, say nothing if you cannot keep it polite.
+- Stand up for one another and take responsibility for each others mistakes.
+- Any team member can request a time-out in team chat, which will suspend any talking in team chat for 30 minutes.
+  - Respect the time out, do not contact each other in DM during this period.
+- We work with a 3 strike system, if you hit strike 3 you will lose your role.
+
+### Rules regarding PRs
+- Any contributor PR requires at least two code reviews, either by developers or trusted community members (at least one developer).
+- Any developer PR requires at least one code review, by another developer.
+- Any PR requires one playtest review, if applicable.
+
+### Guidelines regarding public relations and community management
+
+With an online presence we require maintaining a good public image. For this we have specific guidelines, so each party can take responsibility where necessary.
+
+- Any message* for support or public relations requires a signature**.
+- When sending emails, finish the letter with: Your regards, your name and the OpenRCT2 Team.
+- Please keep a professional tone when communicating for support and public relations purposes.
+
+*Discord messages are already signed.  
+**For the identity associated with each signature refer to the paragraph "Team signatures"
+
+
+#### Team Signatures
+When finding a message on our social media the following signatures refer to the following members:
+
+| Name | Signature |
+| ---- | --------- |
+| Ted (IntelOrca) | ~/i | 
+| Duncan (duncanpumpkin) | ~/d |
+| Michał (janisozaur) | ~/j |
+| Hielke (Broxzier) | ~/b |
+| Tulio (tupaschoal) | ~/t | 
+| Olivier (Oli414) | ~/o |
+| Matthias (ZehMatt) | ~/z |
+| Karst (AuraSpecs)  | ~/a |
+| Michael (mrmbernardi) | ~/m |
```

Comment:
Time to add @Gymnasiast now too!
Also interesting how there are still no conflicts on the first letters

```suggestion
| Michael (mrmbernardi) | ~/m |
| Michael (Gymnasiast) | ~/g |
```